tex/generic/pgf/graphdrawing/lua/pgf/gd/control/doc.lua

```diff
@@ -98,12 +98,12 @@ sense that some nodes are ``nailed to the canvas'' while other
 nodes can ``move freely''.
 ]]
 
---[[
-% TODOsp: codeexamples: the following 3 examples need these libraries
-%    \usetikzlibrary{graphs,graphdrawing}
-%    \usegdlibrary{force}
---]]
-example
+example({options =
+[[
+  preamble=\usetikzlibrary{graphs,graphdrawing}
+    \usegdlibrary{force}
+]],
+code =
```

Comment:
Introduce more code indents? A style similar to the one used in 6a0e08db272f09091ffdfc3f8766c91979138750 could be:
```lua
example ({
  options = [["..."]], -- are the pair of double quotes necessary here?
  code = [["
    ...
  "]]
})
```

Reply:
Reworded